example/serving/grpc/main.go

```diff
@@ -52,8 +52,8 @@ func echoHandler(ctx context.Context, in *common.InvocationEvent) (out *common.C
 		return
 	}
 	log.Printf(
-		"echo - ContentType:%s, Verb:%s, QueryString:%s, %+v",
-		in.ContentType, in.Verb, in.QueryString, string(in.Data),
+		"echo - ContentType:%s, Verb:%s, QueryString:%s, %s",
+		in.ContentType, in.Verb, in.QueryString, in.Data,
```

Comment:
This will print the byte array instead of the data

Reply:
No it want. Tested 

```shell
== APP == 2020/08/18 14:01:33 event - PubsubName:messages, Topic:topic1, ID:3fec7421-d947-41cf-ac9c-3b614b316358, Data: {"to":"Lary","message":"hi","from":"John"}

== APP == 2020/08/18 14:01:33 event - PubsubName:messages, Topic:topic1, ID:1f40a77c-8091-4267-bd5f-ff7bc1835ff8, Data: <message><from>John</from><to>Lary</to></message>

== APP == 2020/08/18 14:01:33 event - PubsubName:messages, Topic:topic1, ID:179f4711-95d6-4558-b6f7-3422f35b4e10, Data: 0x18, 0x2d, 0x44, 0x54, 0xfb, 0x21, 0x09, 0x40
```